src/main/java/training/TableItems.java

```diff
@@ -0,0 +1,28 @@
+package training;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.PutItemOutcome;
+import com.amazonaws.services.dynamodbv2.document.Table;
+
+public class TableItems {
+    public void putItem() {
+
+        AmazonDynamoDB client = AmazonDynamoDBClientBuilder.standard().build();
+        DynamoDB dynamoDB = new DynamoDB(client);
+
+        Table table = dynamoDB.getTable("IGHItems");
+
+        Item item = new Item()
+                .withPrimaryKey("ghid", 1)
+                .withString("Title", "cycle")
+                .withNumber("shelflife", 12)
+                .withBoolean("discontinued", false);
+
+
+        PutItemOutcome outcome = table.putItem(item);

```

Comment:
This variable outcome is not used anywhere.

Reply:
will remove it

Reply:
but if im removing it its not inserting items in table

Reply:
No you should not remove rather try using it as it can tell you if the item inserted properly or not.

Reply:
i removed it because the outcome was not used and instead of this line i used "table.putItem(item);"